projects/packages/scheduled-updates/src/class-scheduled-updates.php

```diff
@@ -169,10 +171,15 @@ public static function run_scheduled_update( ...$plugins ) {
 	 * @param int    $timestamp Timestamp of the first run.
 	 * @param string $interval  Interval of the update.
 	 * @param array  $plugins   List of plugins to update.
-	 * @return \WP_Error|bool True on success, WP_Error on failure.
+	 * @return \WP_Error|true True on success, WP_Error on failure.
 	 */
 	public static function create_scheduled_update( $timestamp, $interval, $plugins ) {
-		return wp_schedule_event( $timestamp, $interval, self::PLUGIN_CRON_HOOK, $plugins, true );
+		$res = wp_schedule_event( $timestamp, $interval, self::PLUGIN_CRON_HOOK, $plugins, true );
```

Comment:
What's "res"? Could we find a more descriptive variable name?

Reply:
Changed, thanks.